components/ChapterTable/styles.module.scss

```diff
@@ -10,20 +10,24 @@
   white-space: nowrap;
   width: 100%;
   overflow-x: scroll;
+  border-bottom: solid 1px lightgray;
+  margin-bottom: 12px;
   .chapter_select:first-child {
     margin-left: 0;
   }
 }
 
 .chapter_select {
-  margin: 12px;
+  margin: 12px 12px 0 12px;
+  background-color: transparent;
+  border: none;
```

Comment:
We could elevate these `border` and `background` resets to a global style for all `button` elements

Reply:
fixed